.github/workflows/sonarcloud.yml

```diff
@@ -9,6 +9,7 @@ jobs:
   sonarcloud:
     name: SonarCloud
     runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.head.repo.fork }}
```

Comment:
I think this should move to line 31. So that we only skip the step using the token. Otherwise we still want to verify it builds.

Reply:
The [CI build](https://github.com/microsoftgraph/msgraph-sdk-go-core/blob/main/.github/workflows/go.yml) verifies builds and runs tests. the only difference with the sonar would be running sonar checks. This means this steps will be a duplication of the GO CI build

Reply:
Thanks for clearing that up. Resolving.